### PR TITLE
please add to schneider_electric.ts

### PR DESCRIPTION
**Pull request by TGruber-HH**
{
        zigbeeModel: ['1GANG/SWITCH/2'],
        model: 'MEG5116-0300_MEG5162-0000',
        vendor: 'Schneider Electric',
        description: 'Merten MEG5162 switch insert (2fold) with Merten System M push button (1fold)',
        extend: [deviceEndpoints({'endpoints': {'left': 1, 'right': 2, 'left_sw': 21}}), identify(),
            onOff({'powerOnBehavior': false, 'endpointNames': ['left', 'right']}),
            commandsOnOff({'endpointNames': ['left_sw']}),
        ],
    };
    {
        zigbeeModel: ['1GANG/ESWITCH/1'],
        model: 'MEG5116-0300_MEG5151-0000',
        vendor: 'Schneider Electric',
        description: 'Merten MEG5151 switch insert with Merten System M push button (1fold)',
        extend: [deviceEndpoints({'endpoints': {'switch': 1, 'switch_sw': 21}}), identify(),
            onOff({'powerOnBehavior': false}),
            commandsOnOff({'endpointNames': ['switch_sw']}),
        ],
    };

for whatever erason, both PR will not be attached

### CODE REVIEW

File: src/devices/schneider_electric.ts [hunk 1]
```diff
@@ -1295,15 +1295,34 @@ const definitions: Definition[] = [
     },
     {
         zigbeeModel: ['2GANG/ESWITCH/2'],
-        model: 'MEG5126-0300/MEG5152-0000',
+        model: 'MEG5126-0300_MEG5152-0000',
```
Comment (Koenkk) [inline src/devices/schneider_electric.ts:1298]: Why was this changed? (and also the `levelConfig` change above)

**Comment by TGruber-HH** [comment]
I thought that name of the device and picture should exactly match. Therefore I changed the slash by the underscore sent via iPhone Thomas GruberAm 18.04.2024 um 22:08 schrieb Koen Kanters ***@***.***>:﻿
@Koenkk commented on this pull request.



In src/devices/schneider_electric.ts:
> @@ -1295,15 +1295,34 @@ const definitions: Definition[] = [
     },
     {
         zigbeeModel: ['2GANG/ESWITCH/2'],
-        model: 'MEG5126-0300/MEG5152-0000',
+        model: 'MEG5126-0300_MEG5152-0000',

Why was this changed? (and also the levelConfig change above)

**Comment by Koenkk** [comment]
Thanks!
